test/acceptance/features/bindAppToService.feature

```diff
@@ -217,3 +217,96 @@ Feature: Bind an application to a service
         And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding Request "binding-request-empty-app" should be changed to "False"
         And jq ".status.conditions[] | select(.type=="InjectionReady").reason" of Service Binding Request "binding-request-empty-app" should be changed to "EmptyApplicationSelector"
 
+    Scenario: Service Binding with Strimzi Kafka Cluster database without applicationSelector
+        Given Imported Nodejs application "nodejs-empty-app" is running
```

Comment:
I think you don't need this step as you don't need an application in this scenario. Removing it will save some execution time.

Reply:
A similar scenario is described here https://github.com/redhat-developer/service-binding-operator/blob/master/test/acceptance/features/bindAppToService.feature#L201

Reply:
@pmacik Can we skip creating a node-js app step where we don't need an application selector?

Reply:
removing this as part of an incoming commit